packages/grafana-data/src/types/icon.ts

```diff
@@ -166,9 +166,12 @@ export const availableIconsIndex = {
   kubernetes: true,
   'layer-group': true,
   'layers-alt': true,
+  'legend-show': true,
+  'legend-hide': true,
```

Comment:
this is really minor, but if this list is alphabetical then `legend-hide` should be before `legend-show`